tag.go

```diff
@@ -0,0 +1,20 @@
+package notmuch
+
+// Copyright © 2015 The go.bindings Authors. Authors can be found in the AUTHORS file.
```

Comment:
I just pushed a patch to master that changes the copyright holder to go.notmuch
globally; would you rebase and fix it here as well?